node/toolrunner.ts

```diff
@@ -1312,18 +1314,20 @@ export class ToolRunner extends events.EventEmitter {
             state.CheckComplete();
         });
 
-        cp.on('exit', (code: number, signal: any) => {
+        cp.on('exit', (code: number, signal: number | NodeJS.Signals) => {
             state.processExitCode = code;
             state.processExited = true;
             this._debug(`Exit code ${code} received from tool '${this.toolPath}'`);
+            this._debug(`Signal ${signal} received from tool '${this.toolPath}'`);
             state.CheckComplete()
         });
 
-        cp.on('close', (code: number, signal: any) => {
+        cp.on('close', (code: number, signal: number | NodeJS.Signals) => {
             state.processExitCode = code;
             state.processExited = true;
             state.processClosed = true;
-            this._debug(`STDIO streams have closed for tool '${this.toolPath}'`)
+            this._debug(`STDIO streams have closed for tool '${this.toolPath}'`);
+            this._debug(`Signal ${signal} received from tool '${this.toolPath}'`);
```

Comment:
Suggestion:
```suggestion
            this._debug(`STDIO streams have closed and received signal ${signal} for tool '${this.toolPath}'`);
```

And the same for others. What do you think?

Reply:
Sure, I'll fix output

Reply:
I've also added exit code to output